auctions/tests/utils_tests.py

```diff
@@ -0,0 +1,21 @@
+import fudge
+
+from django.test import TestCase
+
+from ..utils import issue_state, github_client
+
+
+class IssueStateTest(TestCase):
+
+    def test_return_state(self):
+        url = 'https://github.com/codesy/codesy/issues/158'
+
+        fake_gh_client = (fudge.Fake(github_client)
+                          .returns_fake()
+                          .provides('get_repo')
+                          .returns_fake()
+                          .provides('get_issue')
+                          .returns_fake()
+                          .has_attr(state='open'))
+
+        self.assertEqual('open', issue_state(url, fake_gh_client))
```

Comment:
Great, thanks! You were exactly right - using `expects` allowed me to control the behavior how I wanted.